README.md

```diff
@@ -100,6 +101,7 @@ Now in your termonal do this :
 > - First : "npm install" : to install all npm dependencies
 > - Then type "npm run build"
 > - npm install react-router-dom
+> - npm install @reduxjs/toolkit react-redux
```

Comment:
- [ ] your MIT link is broken, to give your project proper license, kindly fix this

Reply:
I have fixed it !